src/pages/PeoplePage.tsx

```diff
@@ -0,0 +1,48 @@
+import { useState } from 'react';
+import { Loader } from '../components/Loader';
+import { Person } from '../types';
+import { PeopleTable } from '../components/PeopleTable/PeopleTable';
+import { getPeople } from '../api';
+
+export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [loading, setLoading] = useState(true);
```

Comment:
I prefer to use isLoading for boolean types. It's not a fault but just a recommendation